File: programmers/월간_코딩챌린지/약수의갯수와덧셈.py
def solution(left, right):
    total_sum = 0 # 리턴 값
    for num in range(left, right+1):
        div_num = 0  # 약수의 개수
        for i in range(1, num+1):
            if num % i == 0:
                div_num += 1
        total_sum += num * ( -1 if div_num % 2 else 1)
    return total_sum

File: programmers/월간_코딩챌린지/약수의갯수와덧셈.js
function solution(left, right) {
  let total = 0,
    divNum;
  for (let i = left; i <= right; i++) {
    divNum = 0;
    for (let j = 1; j <= i; j++) {
      if (i % j === 0) divNum += 1;
    }
    if (divNum % 2) {
      total -= i;
      continue;
    }
    total += i;
  }
  return total;
}
